dev/com.ibm.ws.concurrency.policy/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -39,7 +39,7 @@ maxWaitForEnqueue=Maximum wait for enqueue
 maxWaitForEnqueue.desc=Specifies the maximum duration of time to wait for enqueuing a task. If unable to enqueue the task within this interval, the task submission is subject to the run-if-queue-full policy. When the maximum wait for enqueue is updated, the update applies only to tasks submitted after that point. Tasks submissions that were already waiting for a queue position continue to wait per the previously configured value.
 
 runIfQueueFull=Run if queue full
-runIfQueueFull.desc=Applies when using the execute or submit methods. Indicates whether or not to run the task on the submitter's thread when the queue is full and the maximum wait for enqueue has been exceeded. When configured to false, the task submission is rejected after the maximum wait for enqueue elapses instead of running on the submitter's thread.
+runIfQueueFull.desc=Applies when using the execute or submit methods. Indicates whether or not to run the task on the submitter's thread when the queue is full and the maximum wait for enqueue has been exceeded. If the maximum policy is configured to strict, the ability to run on the submitter's thread is additionally contingent on the maximum concurrency constraint. If the task can not run on the submitter's thread, the task submission is rejected after the maximum wait for enqueue elapses.
 
```

Comment:
Hi! Recommend: Applies when you use the......
Also, should we write `execute` or `submit` in code phrases, as I did here? If so, I recommend that change.
Recommend: enqueue _was_ exceeded
Please change can not to "cannot"
Otherwise, looks good.

Reply:
I'll make the last 2 updates.  However, regarding execute and submit, we are referring to the names of methods on the java.util.concurrent.ExecutorService API.  We cannot rename them, and referring to them by any other terms within the message will cause confusion.  The term start is not even accurate here, as the fact that the submit or execute method completes does not mean the task will necessarily ever start.  The task could be canceled or abort before that point.  I'm interested in the possibility you mentioned of using "code phrases" to distinguish these as method names if that is possible.

Reply:
Nathan can you use <>. In .dita ours is <codeph>, and as Laura mentioned in .md it is `phrase here`. I see <> in pii messages, as well. But it tells users and translators that it is a name of something or to look for something with that phrase.

Reply:
If you choose to use execute, that is fine. IBM style manual recommends not using it, therefore, we try to look for other ways when we can.

Reply:
@bcpswope We don't have any choice in the matter of including the method name, execute.  We didn't come up with that name ourselves; it is simply what the java.util.concurrent.ExecutorService API defines as the method name, and we must use it.  I have added the less-than/greater-than symbols around the method names as requested.  I did have some reservations this might cause the enclosed text not to render at all in browsers due to appearance as HTML tags.
I have also made the other requested updates - fixing the typo in "cannot" and replacing "has been" with "was".